### PR TITLE
10.1.0 release

### LINKED ISSUE #3684: Intermediate release so ReactiveCocoa builds on Xcode 11
There's already a PR merged in to master which updates the dependencies to support building with Xcode 11, it would be nice to create a `10.0.1` for everyone to start using rather than using the commit hash.

### PR DESCRIPTION
**Pull request by ikesyo**
Closes #3684.

#### Checklist
- [x] Updated CHANGELOG.md.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
       script:
       - pod repo update
       - pod lib lint ReactiveCocoa.podspec --use-libraries
-      - pod lib lint ReactiveMapKit.podspec --use-libraries
+      - pod lib lint ReactiveMapKit.podspec --use-libraries --include-podspecs=ReactiveCocoa.podspec
```
Comment (ikesyo) [inline .travis.yml]: https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3656#issuecomment-477401200

**Comment by Lewion** [comment]
You probably want to do: 
`pod lib lint --include-podspecs=\{ReactiveCocoa.podspec\} ReactiveMapKit.podspec`

**Comment by Lewion** [comment]
> You probably want to do:
> `pod lib lint --include-podspecs=\{ReactiveCocoa.podspec\} ReactiveMapKit.podspec`

That or upgrade cocoapods on travis. It seems your version is missing `include-podspecs`

**Comment by ikesyo** [comment]
I'll go ahead. :shipit:

**Pull request merged**